jest.config.js

```diff
@@ -27,14 +27,23 @@ module.exports = {
     'website/.*',
     'e2e/runtime-internal-module-registry/__mocks__',
   ],
-  projects: ['<rootDir>', '<rootDir>/examples/*/'],
+  projects: [
+    '<rootDir>',
+    '<rootDir>/examples/*/',
+    {
+      displayName: 'typings test',
```

Comment:
can we do something like this to make it be more "typescript color" than green?

```suggestion
      displayName: {
        color: 'blue',
        name: 'typings test',
      },
```